test/topology_experimental_raft/test_tablets.py

```diff
@@ -1430,3 +1435,31 @@ async def assert_empty_table():
 
     logger.info("Verify data is not resurrected")
     await assert_empty_table()
+
+@pytest.mark.repair
+@pytest.mark.asyncio
+async def test_tablet_manual_repair(manager: ManagerClient):
+    servers, cql, hosts, table_id = await create_table_insert_data_for_repair(manager, fast_stats_refresh=False)
+    token = -1
+
+    start = time.time()
+    await manager.api.tablet_repair(servers[0].ip_addr, "test", "test", token)
+    duration = time.time() - start
+    map1 = await load_tablet_repair_time(cql, hosts[0:1], table_id)
+    logging.info(f'map1={map1} duration={duration}')
+
+    await asyncio.sleep(1)
+
+    start = time.time()
+    await manager.api.tablet_repair(servers[0].ip_addr, "test", "test", token)
+    duration = time.time() - start
+    map2 = await load_tablet_repair_time(cql, hosts[0:1], table_id)
+    logging.info(f'map2={map2} duration={duration}')
+
+    t1 = map1[str(token)]
+    t2 = map2[str(token)]
+    logging.info(f't1={t1} t2={t2}')
+
+    assert t2 > t1
+
+    await cql.run_async("DROP KEYSPACE test;")
```

Comment:
Not needed. Makes the test slower, so better avoid.

Reply:
ok